docs/reference/platforms/android/gradle.rst

```diff
@@ -266,6 +274,14 @@ will result in an ``AndroidManifest.xml`` declaration of::
 
     <uses-permission android:name="android.permission.HIGH_SAMPLING_RATE_SENSORS">
 
+``target_sdk_version``
+----------------------
+
+The SDK version that the app will target. This controls the APIs that are available for
+use in your application. This is *not* the Android version; it is the underlying SDK
```

Comment:
Technically the `compileSdkVersion` controls the available APIs, while `targetSdkVersion` controls various [behavior and compatibility changes](https://developer.android.com/guide/topics/manifest/uses-sdk-element#target). The latter is usually more important, since in most cases your available APIs are constrained by the `minSdkVersion` anyway.

Reply:
I've clarified the text to match the behaviour of the two settings. I've also mentioned the explicit settings (both here, and in the iOS/macOS settings).